python_modules/dagster/dagster/_core/definitions/freshness_checks/last_update.py

```diff
@@ -100,10 +135,169 @@ def build_last_update_freshness_checks(
         AssetChecksDefinition: An `AssetChecksDefinition` object, which can execute a freshness check
             for all provided assets.
     """
-    return build_freshness_checks_for_assets(
-        assets=assets,
+    check.inst_param(lower_bound_delta, "lower_bound_delta", datetime.timedelta)
+    check.str_param(timezone, "timezone")
+    check.opt_str_param(deadline_cron, "deadline_cron")
+    check.invariant(
+        deadline_cron is None or is_valid_cron_string(deadline_cron), "Invalid cron string."
+    )
+    check.inst_param(severity, "severity", AssetCheckSeverity)
+    check.sequence_param(assets, "assets")
+    ensure_no_duplicate_assets(assets)
+    return _build_freshness_multi_check(
+        asset_keys=assets_to_keys(assets),
         deadline_cron=deadline_cron,
         timezone=timezone,
         severity=severity,
         lower_bound_delta=lower_bound_delta,
     )
+
+
+def _build_freshness_multi_check(
+    asset_keys: Sequence[AssetKey],
+    deadline_cron: Optional[str],
+    timezone: str,
+    severity: AssetCheckSeverity,
+    lower_bound_delta: datetime.timedelta,
+) -> AssetChecksDefinition:
+    params_metadata: dict[str, Any] = {TIMEZONE_PARAM_KEY: timezone}
+    if deadline_cron:
+        params_metadata[DEADLINE_CRON_PARAM_KEY] = deadline_cron
+    if lower_bound_delta:
```

Comment:
This will always be specified, right?

Reply:
that's right. Artifact from the old implementation.